docs/pages/docs/usage/messages.mdx

```diff
@@ -278,3 +278,62 @@ Messages are always parsed and therefore e.g. for rich text you need to supply t
 </Callout>
 
 The value of a raw message can be any valid JSON value: strings, booleans, objects and arrays.
+
+## Right-to-left languages
+
+Languages such as Arabic, Hebrew and Persian use [right-to-left script](https://en.wikipedia.org/wiki/Right-to-left_script) (often abbreviated as RTL). For these languages, writing begins on the right side of the page and continues to the left.
```

Comment:
Would it be beneficial to include a textual example in order to enhance clarity?
Like:
```txt
النص في اللغة العربية _مثلا_ يُقرأ من اليمين لليسار
```

Just to show the problem that we are solving

Reply:
Good idea, thanks! I was quite amazed that in the editor even the cursor keys immediately switch once you enter a RTL script!